receiver/awsxrayreceiver/internal/translator/translator_test.go

```diff
@@ -880,7 +880,7 @@ func TestTranslation(t *testing.T) {
 				)
 			}
 
-			traces, totalSpanCount, err := ToTraces(content)
+			traces, totalSpanCount, err := ToTraces(content, nil)
```

Comment:
This should use a test recorder to validate the interaction with the recorder.

Reply:
Added test recorder and validation.